server/src/internalClusterTest/java/org/opensearch/search/suggest/CompletionSuggestSearchIT.java

```diff
@@ -1252,7 +1288,7 @@ private static void assertSuggestions(
 
     private void createIndexAndMappingAndSettings(Settings settings, CompletionMappingBuilder completionMappingBuilder) throws IOException {
         XContentBuilder mapping = jsonBuilder().startObject()
-            .startObject(TYPE)
+            .startObject(MapperService.SINGLE_MAPPING_NAME)
```

Comment:
It's not! The type isn't used in the mappings at all so this can go away. I was quickly ripping through the API changes to keep the blast radius manageable and planned to come back and clean up the mappings in one of the follow on mapping type removal PRs. 

But since you asked :wink: I fixed this one.